### PR TITLE
Added cooldown command

### PR DESCRIPTION
**Pull request by davfsa**
# Please check the following boxes

- [x] I've read and agree to the [Contribution Guidelines](https://github.com/Dougley/MBv2/blob/master/.github/CONTRIBUTING.md) and to the [Code of Conduct](https://github.com/Dougley/MBv2/blob/master/.github/CODE_OF_CONDUCT.md)
- [x] I've checked that I didn't commit sensitive information like tokens or other login information
- [x] I haven't hardcoded any IDs or anything else that shouldn't be (with the exception being permissions)
- [x] I tested my code and I verified it's working to the best of my ability (Don't have the ability to do that atm.... but Dougley said he could do it)
- [x] I checked if my code doesn't violate the styleguide with `npm test` (Travis CI Helped)

# Describe your pull request
This pull request will add a `cooldown` command that will let the custodian know what his xp limits are (this idea was taken from EvilDabbit, in the Discord Testers Server)

### CODE REVIEW

**Comment by Dougley** [comment]
Did you test this? At first glance it looks good.

File: src/commands/cooldown.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      color: 0xa09edd,
+      footer: {
+        icon_url: global.bot.user.dynamicAvatarURL('png', 32),
+        text: `MegaBot ${process.env.NODE_ENV === 'debug' ? 'Development version' : 'v' + require('../../package').version}`
+      },
+      thumbnail: {
+        url: `https://cdn.discordapp.com/avatars/${userdata.id}/${userdata.avatar}.png`
+      },
+      fields: [
+        {
```
Comment (Dougley) [inline src/commands/cooldown.js]: These fields would work better inline

File: src/commands/cooldown.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    level: 0,
+    alias: ['caps']
+  },
+  fn: (msg) => {
+    const data = db.getUser(msg.author.id)
+    msg.author.getDMChannel().then(async c => {
+      if (data.entitlements.includes('fake-stats')) msg.channel.createMessage(`<@${msg.author.id}>, an unexpected error occurred while getting your stats, try again later.`)
+      await c.createMessage(generateEmbed(msg.author, data))
+      if (msg.channel.guild) return msg.delete()
+      if (!data.entitlements.includes('fake-stats')) await msg.addReaction(`${ids.emojis.confirm.name}:${ids.emojis.confirm.id}`)
```
Comment (Dougley) [inline src/commands/cooldown.js]:
```suggestion
      else await msg.addReaction(`${ids.emojis.confirm.name}:${ids.emojis.confirm.id}`)
```

File: src/commands/cooldown.js (hunk 1 above)
Comment (davfsa) [reply src/commands/cooldown.js]: Fixed it

File: src/commands/cooldown.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+module.exports = {
+  meta: {
+    level: 0,
+    alias: ['caps']
+  },
+  fn: (msg) => {
+    const data = db.getUser(msg.author.id)
+    msg.author.getDMChannel().then(async c => {
+      if (data.entitlements.includes('fake-stats')) msg.channel.createMessage(`<@${msg.author.id}>, an unexpected error occurred while getting your stats, try again later.`)
```
[the remark this answers is not in the document]
Comment (davfsa) [reply src/commands/cooldown.js]: Removed it

**Pull request merged**